lib_xua/lib_build_info.cmake

```diff
@@ -20,14 +20,14 @@ set(LIB_INCLUDES api
                  src/hid
                  src/midi)
 set(LIB_OPTIONAL_HEADERS xua_conf.h static_hid_report.h)
-set(LIB_DEPENDENT_MODULES "lib_locks"
-                          "lib_logging"
-                          "lib_mic_array(feature/xcommon_cmake)"
+set(LIB_DEPENDENT_MODULES "lib_adat(1.1.0)"
+                          "lib_locks(2.2.0)"
+                          "lib_logging(3.2.0)"
+                          "lib_mic_array(4.6.0)"
                           "lib_spdif"
```

Comment:
this is v5.0.0 in module_build_info

Reply:
Will update both to 6.x.y when lib_spdif is tagged for the upcoming release.